lib/node_modules/@stdlib/math/base/special/exp/package.json

```
@@ -66,5 +66,64 @@
     "euler",
     "power",
     "number"
-  ]
+  ],
+  "__stdlib__": {
+    "$schema": "math/base@v1.0",
+    "base_alias": "exp",
+    "alias": "exp",
+    "pkg_desc": "Natural exponential function",
+    "desc": "Evaluates the natural exponential function.",
```

```suggestion
    "pkg_desc": "evaluate the natural exponential function",
    "desc": "evaluates the natural exponential function",
```

Keep lowercase so downstream tools don't have to perform extra processing.